pkg/k8scontext/context.go

```diff
@@ -231,6 +262,9 @@ func (i *InformerCollection) Run(stopCh chan struct{}) {
 
 	go i.Ingress.Run(stopCh)
 
+	go i.AzureIngressManagedLocation.Run(stopCh)
```

Comment:
we should run wait until these are synced as well ?
cache.WaitForCacheSync(stopCh, i.AzureIngressManagedLocation.HasSynced)
cache.WaitForCacheSync(stopCh, i.AzureIngressProhibitedLocation.HasSynced)
so that we only move once we know what we can modify ?

Reply:
Yes!  Thanks for catching that!!
Let me make this a bit more robust!